tools/ci_build/github/linux/run_build.sh

```diff
@@ -26,14 +26,13 @@ if [ $BUILD_DEVICE = "gpu" ]; then
 elif [ $BUILD_DEVICE = "tensorrt" ]; then
```

Comment:
does this mean for cpu and gpu providers the test is run twice, once in run_build.sh
and another time when build.py is invoked?
would it be easier if we add it to run_build.sh only instead?

Reply:
The line /home/onnxruntimedev/Release/onnx_test_runner -e ... /data/onnx is not needed according to @snnn